.github/workflows/release.yml

```diff
@@ -15,9 +15,9 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: actions/setup-go@v2
+      - uses: actions/setup-go@v3
         with:
-          go-version: '1.17'
+          go-version: '1.18'
```

Comment:
the SDK is targeting go 1.17, why use a different version?

Reply:
the SDK is targeting go 1.18

https://github.com/cosmos/cosmos-sdk/blob/b363e0172ed4fdba3effd349c33fe2f48f2f5846/go.mod#L1

Reply:
https://github.com/cosmos/cosmos-sdk/pull/11706

Reply:
https://github.com/cosmos/cosmos-sdk/pull/11663

Reply:
that said, this seems to work fine with go 1.17 so I'll leave it to this repository's maintainer's opinion re: weather or not to use 1.18